server/src/models/Sandwich.js

```diff
@@ -24,6 +24,22 @@ class Sandwich extends unique(Model) {
     }
   }
 
+  async averageRating() {
+    let sum = 0
+    let average = 0
+    const reviews = await this.$relatedQuery("reviews")
+    if (reviews.length !== 0) {
+      const ratingsArray = reviews.map(review => {
+        return review.starRating
+      })
+      ratingsArray.forEach(rating => {
+        sum = sum + rating
+      })
+      average = (sum/ratingsArray.length)
+    } else average = 0
```

Comment:
you dont need an else. average will already equal zero if the `if` condition is not true. remove the else.